### PR TITLE
Update Prereqs, Troubleshooting, Resources, Next step

### PR DESCRIPTION
**Pull request by everett-toews**
@carolynvs Please review these changes to see if it still makes sense with a more Carina-centric view.

I also have some questions around your use of hosts vs clusters in Connect containers by using the ambassador pattern. We can chat tomorrow.

### CODE REVIEW

File: _tutorials/030-connect-docker-containers-with-links.md [hunk 1]
```diff
@@ -139,27 +132,26 @@ and how the containers use it to communicate.
...
-7. Log out of the web container.
+1. Log out of the web container.
 
     ```bash
     $ exit
     ```
 
-8. Log in to the app container by running the following command. Depending on your local configuration
-    you might need to use a workarounds from [Error running interactive Docker shell on Windows][tty-workaround].
+1. Log in to the app container by running the following command.
```
Comment (carolynvs) [inline _tutorials/030-connect-docker-containers-with-links.md]: We shouldn't drop the warning about running that command on Windows.

File: _tutorials/030-connect-docker-containers-with-links.md [hunk 2]
```diff
@@ -187,20 +179,28 @@ and how the containers use it to communicate.
...
 
-12. Log out of the app container.
+1. Log out of the app container.
 
     ```bash
     $ exit
     ```
 
-[tty-workaround]: {{site.baseurl}}/docs/references/troubleshooting-cannot-enable-tty-mode-on-windows/
+### Troubleshooting
```
Comment (carolynvs) [inline _tutorials/030-connect-docker-containers-with-links.md]: I feel a bit jerky moving the warning about running a command that is known to not work on windows down here and hoping they keep reading for the troubleshooting section.

File: _tutorials/031-connect-docker-containers-ambassador-pattern.md [hunk 3]
```diff
@@ -13,22 +13,17 @@ topics:
 ---
 
 This tutorial demonstrates how to connect Docker containers by using the ambassador pattern
-so that they can communicate over the network and across Docker hosts. For information about
-the ambassador pattern, see [Docker networking basics][networking-basics].
+so that they can communicate over the network and across Docker hosts.
```
Comment (carolynvs) [inline _tutorials/031-connect-docker-containers-ambassador-pattern.md]: This is undoing feedback that I was directly asked in include (a link to what heck the ambassador pattern is). Is the idea that if they don't know, they won't be on this page? Or that they will scroll down to the resources and figure it out?

File: _tutorials/031-connect-docker-containers-ambassador-pattern.md [hunk 4]
```diff
@@ -13,22 +13,17 @@ topics:
...
 
 ### Prerequisite
 
-Two Docker hosts using [Linux][docker-linux], [Docker Toolbox][docker-toolbox], or [Carina][carina]
-
-[docker-linux]: http://docs.docker.com/linux/step_one/
-[docker-toolbox]: https://www.docker.com/toolbox
-[carina]: http://app.getcarina.com/
+[Create and connect to a cluster](/docs/tutorials/create-connect-cluster/) named clustera and a cluster named clusterb
```
Comment (carolynvs) [inline _tutorials/031-connect-docker-containers-ambassador-pattern.md]: I think this needs a period at the end as it's a sentence.

File: _tutorials/031-connect-docker-containers-ambassador-pattern.md [hunk 5]
```diff
@@ -124,12 +120,22 @@ You now have two containers that can communicate across Docker hosts.
 
 ![Docker ambassador pattern topology]({% asset_path connect-docker-containers-ambassador-pattern/ambassador-pattern-topology.svg %})
 
+### Troubleshooting
+
+See [Error running interactive Docker shell on Windows](/docs/references/troubleshooting-cannot-enable-tty-mode-on-windows/).
```
Comment (carolynvs) [inline _tutorials/031-connect-docker-containers-ambassador-pattern.md]: This isn't needed as we aren't asking them to run `docker exec -it`

**Comment by carolynvs** [comment]
@everett-toews Thanks for poking me about this, I missed the original GH notification. :smile:  I have finished adding my comments.

File: _tutorials/030-connect-docker-containers-with-links.md (hunk 1 above)
Comment (everett-toews) [inline _tutorials/030-connect-docker-containers-with-links.md]: Added it back.

File: _tutorials/030-connect-docker-containers-with-links.md (hunk 2 above)
Comment (everett-toews) [inline _tutorials/030-connect-docker-containers-with-links.md]: It's back

File: _tutorials/031-connect-docker-containers-ambassador-pattern.md (hunk 3 above)
Comment (everett-toews) [inline _tutorials/031-connect-docker-containers-ambassador-pattern.md]: See this [comment by me](https://github.com/rackerlabs/docs-container-service/pull/274#issuecomment-150612235) and the next comment by Kelly for the thinking behind the preference not to link.

However, I don't know the context behind the feedback so I'll default to putting the link back in.

File: _tutorials/031-connect-docker-containers-ambassador-pattern.md (hunk 5 above)
Comment (everett-toews) [inline _tutorials/031-connect-docker-containers-ambassador-pattern.md]: Removed.

**Pull request merged**